### ISSUE TITLE
Aarch64/ARM64  for GNU/Linux?

### ISSUE THREAD

**Issue by mzs114** [netbirdio/netbird#610]
**Is your feature request related to a problem? Please describe.**
Hi, I am trying to install netbird clients on ARMhf and ARM64 or as Debian and Raspbian call it - Aarch64

**Describe the solution you'd like**
Install netbird and netbird-ui on ARM 32 and 64 bit.

**Describe alternatives you've considered**
There are no alternatives other than using x64 hardware, meaning stop using Raspberry PI and AWS Gravition instances

**Additional context**
ARM is being used on user end devices like Raspberry Pi and on enterprise infra mainly via cloud vendors, having netbird on ARM would be a great option. 
Next, not sure what is the current approach, but do consider using [SUSE OBS](https://openbuildservice.org/) service for publishing these native packages, to make it easier.

**Comment by mlsmaycon** [comment]
Hello @mzs114, thanks for reporting the issue.

To clarify, are you getting an error when following the Ubuntu installation steps?

Currently we are releasing packaging for both architectures.

**Comment by mzs114** [comment]
Yes,  I thought it is due to unsupported architecture, but upon checking the `/var/log/netbird/client.log`, I found below:

`time="2022-12-06T21:51:55+05:30" level=error msg="failed creating tunnel interface wt0: [no such device]" file="engine.go:251"
time="2022-12-06T21:51:55+05:30" level=error msg="error while starting Netbird Connection Engine: no such device" file="connect.go:151"
time="2022-12-06T21:52:06+05:30" level=info msg="using userspace WireGuard" file="iface_linux.go:27"
time="2022-12-06T21:52:06+05:30" level=error msg="failed creating tunnel interface wt0: [no such device]" file="engine.go:251"
time="2022-12-06T21:52:06+05:30" level=error msg="error while starting Netbird Connection Engine: no such device" file="connect.go:151"`

**Comment by mlsmaycon** [comment]
Hello, @mzs114 are you running on a container? 

If yes, can you review the docker run command if it includes the `--cap-add=NET_ADMIN` flag?

If not, can you run:
```
ls -l /dev/net/tun
```

**Comment by mzs114** [comment]
It is on Raspberry Pi 11 OS and not on a container:

```
uname -a
Linux raspberrypi 5.15.61-v7+ #1579 SMP Fri Aug 26 11:10:59 BST 2022 armv7l GNU/Linux

ls -l /dev/net/tun
crw-rw-rw- 1 root root 10, 200 Sep 21 21:18 /dev/net/tun

dpkg -l wireguard
||/ Name           Version        Architecture Description
+++-==============-==============-============-====================================================
ii  wireguard      1.0.20210223-1 all          fast, modern, secure kernel VPN tunnel (metapackage)

```
